contracts/DAORegistryScheme.sol

```diff
@@ -0,0 +1,181 @@
+pragma solidity >=0.5.4 <0.6.0;
+
+import "@daostack/arc/contracts/universalSchemes/UniversalScheme.sol";
+import "@daostack/arc/contracts/votingMachines/VotingMachineCallbacks.sol";
+import "@daostack/infra/contracts/votingMachines/IntVoteInterface.sol";
+import "@daostack/infra/contracts/votingMachines/VotingMachineCallbacksInterface.sol";
+import "./DAORegistry.sol";
+
+/**
+ * @title A universal scheme for letting organizations manage a registrar of named DAOs
+ * @dev The DAORegistryScheme has a registry of DAOs for each organization.
+ *      The organizations can through a vote choose to register allowing them to add/remove names inside the registry.
+ */
+contract DAORegistryScheme is UniversalScheme, VotingMachineCallbacks, ProposalExecuteInterface {
+
+    event NewDAOProposal (
+        address indexed _avatar,
+        bytes32 indexed _proposalId,
+        address _proposedAvatarAddress,
+        string _proposedAvatarName
+    );
+
+    event RemoveDAOProposal (
+        address indexed _avatar,
+        bytes32 indexed _proposalId,
+        address _proposedAvatarAddress
+    );
+
+    event ProposalExecuted(address indexed _avatar, bytes32 indexed _proposalId, int256 _param, bytes _returnValue);
+    event ProposalDeleted(address indexed _avatar, bytes32 indexed _proposalId);
+
+    // a DAORegistryProposal is a proposal to add or remove a named DAO to/from an organization
+    struct DAORegistryProposal {
+        bytes callData; //The abi encode data for the call to registry contracts
+        uint256 value;
+    }
+
+    // A mapping from the organization (Avatar) address to the saved data of the organization:
+    mapping(address=>mapping(bytes32=>DAORegistryProposal)) public organizationsProposals;
+
+    // A mapping from hashes to parameters (use to store a particular configuration on the controller)
+    struct Parameters {
+        bytes32 voteParams;
+        IntVoteInterface intVote;
+        DAORegistry daoRegistry;
+    }
+
+    mapping(bytes32=>Parameters) public parameters;
```

Comment:
Please use a more descriptive name for this. You may just as well call this `stuff` or `things`... :-) What kind of paramaters are these? Of what?

Reply:
I see your point. I thought this was a convention since all other universal schemes have it this way :P